### PR TITLE
test: setup e2e tests

### PR DESCRIPTION
**Pull request by 0xmad**
- [x] Add playwright
- [x] Add ci job
- [x] Add connect wallet test
- [x] Remove type module

Note: sometimes due to rpc timeouts, some tests can fail but it can be fixed with rerun.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +74,8 @@
     "@commitlint/cli": "^19.3.0",
     "@commitlint/config-conventional": "^19.2.2",
     "@next/eslint-plugin-next": "^14.2.3",
+    "@playwright/test": "^1.45.0",
```
Comment (ctrlc03) [inline package.json:77]: would they need to be in dependencies or devDependencies is fine too?

**Review by ctrlc03 — COMMENTED** [review]
Nice, thank you, just left 2 comments/questions

File: .env.example [hunk 2]
```diff
@@ -92,9 +91,6 @@ NEXT_PUBLIC_MACI_START_BLOCK=
 
 NEXT_PUBLIC_MACI_SUBGRAPH_URL=
 
-# Vercel Store token to store the application images
-BLOB_READ_WRITE_TOKEN=""
```
[the remark this answers is not in the document]
Comment (0xmad) [reply .env.example:96]: It's duplicated key in .env.example, so I just removed second one.

File: package.json (hunk 1 above)
Comment (0xmad) [reply package.json:77]: It's dev dependency because we don't need in the bundle

Comment (ctrlc03) [reply package.json:77]: oh right yea I thought it was inside dependencies, all good then

**Review by ctrlc03 — APPROVED** [review]
Thanks

**Pull request merged**